config/monitoring/hive_clustersync_servicemonitor.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: monitoring.coreos.com/v1
+kind: ServiceMonitor
+metadata:
+  name: hive-clustersync
+spec:
+  endpoints:
+  - interval: 30s
+    path: /metrics
+    port: metrics
+    scheme: http
```

Comment:
I've received some grief in the past for exporting metrics over http on CCO. If we're sure it is acceptable to export metrics over http then disregard this.

Reply:
This setup is as-is by service delivery currently, so it seems like it has been acceptable.

running metrics on https would need more work in the codebase itself, and we can add that if a requirement comes in.